packages/projects-docs/pages/learn/repositories/devtools.mdx

```diff
@@ -4,56 +4,42 @@ description:
 ---
 
 import { Callout } from 'nextra-theme-docs'
+import Video from '../../../../../shared-components/Video'
 
 # DevTools
  
-![CodeSandbox Preview](../images/devtools-open.jpg)
+ DevTools allow you to work faster and more efficient. They are useful for debugging, testing, and developing. They are collaborative by default, allowing you to share their content: a terminal output, a preview error, a console log...
 
-Devtools are tools for developers to work faster and more efficiently. They are useful for debugging, testing, and developing. They are collaborative by default, allowing you to share their content: a terminal output, a preview error, a console log...
-
-Add as many devtools as you want. For one or two devtools, the layout will adapt to the viewport; more than three items will create a scroll. 
-
-
-## Start a Devtool
-
-You can add a Devtool by clicking on the `+` button on the header or by pressing <kbd>Cmd/Clt</kbd> <kbd>shift</kbd> <kbd>o</kbd> to open the DevTool menu.
-
-
-
-## Manage your DevTool 
-
-![CodeSandbox Preview](../images/devtools-options.jpg)
-
-#### Minimize
-
-Minimize the Devtool by clicking on the `...` icon in the top left corner and selecting the option `Minimize`. A minimized view is useful when you have limited viewport space and would like to still track some processes. 
+ 
+<Video src="../../devtools-management.mp4" />
 
-#### Close 
+Add as many DevTools as you want. Each one lives in its own tab and each tab can be moved around, reordered or dropped on to another tab to split the view.
 
-Close the Devtool by clicking on the `...` in the top left corner and selecting the option `Close`. You can also close all devtools through the command palette (<kbd>Cmd/Clt</kbd> <kbd>K</kbd>) by typing `close all devtools`.
+When a tab is closed, the layout will re-arrange to fill up all the available space. Your layout is persisted in the browser, so you will see the same DevTools when you come back later to your VM.
 
 <Callout emoji="→">
-Closing a devtool will not stop it, it will keep running in the background. If you want to stop the process, you must select the option `Close and stop`. 
+Closing a devtool will not stop it, it will keep running in the background. If you want to stop the process, you must select the option `Close and stop` from the dropdown menu `...`. 
```

Comment:
```suggestion
Closing a DevTool will not stop it—it will keep running in the background. If you want to stop the process, you must select the option `Close and stop` from the dropdown menu `...`. 
```